ui/app/src/components/GlobalApp/GlobalApp.tsx

```diff
@@ -52,11 +60,79 @@ interface GlobalAppProps {
   footerHtml: string;
 }
 
+export function GlobalAppRouterProvider(props: GlobalAppProps) {
+  const { passwordRequirementsMinComplexity } = props;
+  const globalNavigation = useGlobalNavigation();
+  const { classes } = useStyles();
+
+  const router = createHashRouter(
+    createRoutesFromElements(
+      <Route path="/" element={<GlobalApp {...props} />}>
```

Comment:
Let's create some sort of enum with the routes so we can use them across the various places. For example here and when using `navigate('/global-config') => navigate(Routes.GlobalConfig)`